### PR TITLE
solution updated cisco aci

### PR DESCRIPTION
**Pull request by v-laanjana**
Change(s):
See guidance below
Reason for Change(s):
This PR is part of template spec solution migration for solution Illusive Attack Management System.
See guidance below
Version Updated:
2.0.0
See guidance below
Testing Completed:
The link for test results is
[Sentinel Solution Migratio](https://microsoftapc.sharepoint.com/:w:/r/teams/SentinelCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7B41490108-E7DD-4741-A24C-63F7D528B195%7D&file=Sentinel%20Solution%20Migration%20-TemplateSpec%20-%20cisco%20aci.docx&action=default&mobileredirect=true&share=IQEIAUlB3edBR6JMY_fVKLGVAWUGNyDw5LnboHSH4rBBdAw)n

See guidance below
This is an 3P solution.

### CODE REVIEW

**Comment by v-sabiraj** [comment]
Please remove Preview for the solution
image

**Comment by v-sabiraj** [comment]
image
please change text Azure Sentinel to Microsoft Sentinel.

**Comment by v-sabiraj** [comment]
@v-laanjana, please change availability to false, thanks.
image

File: Solutions/Cisco ACI/Package/mainTemplate.json [hunk 1]
```diff
@@ -0,0 +1,657 @@
...
+    "workspace": {
+      "defaultValue": "",
+      "type": "string",
+      "metadata": {
+        "description": "Workspace name for Log Analytics where Microsoft Sentinel is setup"
+      }
+    }
+  },
+  "variables": {
+    "solutionId": "azuresentinel.azure-sentinel-solution-ciscoaci-preview",
```
Comment (v-sabiraj) [inline Solutions/Cisco ACI/Package/mainTemplate.json:33]: @v-laanjana, please remove -preview from solutionId

**Comment by v-sabiraj** [comment]
image
Please replace data connector with solution

File: Solutions/Cisco ACI/Package/mainTemplate.json (hunk 1 above)
Comment (v-laanjana) [reply Solutions/Cisco ACI/Package/mainTemplate.json:33]: done

**Comment by v-sabiraj** [comment]
@v-laanjana, can you please update azure sentinel to Microsoft sentinel in mainTemplate also. Thanks.
image

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**